pages/about.html

```diff
@@ -104,7 +106,9 @@ <h3>Is Web-Dev Right For Me?</h3>
     </div>
   </section>
   <footer class="footer">
-    <div class="logo"></div>
+    <a href="../index.html">
```

Comment:
- [ ] [OPTIONAL] The file path to the logo can be a `#` or just `index.html` to reference its self